File: mobile-client/ios/Contract/Identity/Session/SessionIdentityPayload.swift
public struct SessionIdentityPayload {
    public let vendorId: String?
    public let accountId: String?
    public let displayName: String?
    public let activeRole: String?
    public let authenticated: Bool
    public let emailVerified: Bool
    public let secondFactorEnabled: Bool

    public init(vendorId: String?, accountId: String?, displayName: String?, activeRole: String?, authenticated: Bool, emailVerified: Bool, secondFactorEnabled: Bool) {
        self.vendorId = vendorId
        self.accountId = accountId
        self.displayName = displayName
        self.activeRole = activeRole
        self.authenticated = authenticated
        self.emailVerified = emailVerified
        self.secondFactorEnabled = secondFactorEnabled
    }
}

File: client/android/Contract/Identity/Session/SessionIdentityPayload.kt
package app.mobiling.client.contract.identity.session

data class SessionIdentityPayload(
    val vendorId: String?,
    val accountId: String?,
    val displayName: String?,
    val activeRole: String?,
    val authenticated: Boolean,
    val emailVerified: Boolean,
    val secondFactorEnabled: Boolean,
)
